src/pytesseract.py

```diff
@@ -108,7 +147,7 @@ def run_tesseract(input_filename,
     if extension != 'box':
         command.append(extension)
 
-    proc = subprocess.Popen(command, stderr=subprocess.PIPE)
+    proc = subprocess.Popen(command, **subprocess_args(True))
```

Comment:
Passing the value True, seems to be redundant, since we have default value for the argument (which also defaults to True).